softdevice_controller/include/sdc_hci_cmd_controller_baseband.h

```diff
@@ -0,0 +1,587 @@
+/*
```

Comment:
should the header file name be the same as the abbreviation?

`sdc_hci_cmd_cb.h`?